pkgs/applications/misc/keepass-plugins/keepassotpkeyprov/default.nix

```
@@ -0,0 +1,31 @@
+{ stdenv, buildEnv, fetchzip, mono }:
+
+let
+  version = "2.6";
+  drv = stdenv.mkDerivation {
+    name = "keepassotpkeyprov-${version}";
```

```suggestion
    pname = "otpkeyprov";
    inherit version;
```

The plugin is listed as just `otpkeyprov` on KeePass website.
And the resulting attribute name [as listed on NixOS.org](https://nixos.org/nixos/packages.html?channel=nixpkgs-unstable&query=keepass-k) will then be `keepass-otpkeyprov`

---

done